example/storybook/stories/CustomScreenInjection/CustomScreenInjection.stories.tsx

```diff
@@ -14,14 +14,16 @@ import {
   UsersScreen,
 } from './customScreens';
 import { Navigation } from '@lifeomic/chromicons-native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
 storiesOf('Custom Screen Injection', module)
   .addDecorator(withKnobs)
   .add('Default', () => {
     return (
       <DeveloperConfigProvider
         developerConfig={{
-          getAdditionalHomeScreens: (HomeStack) => {
+          getAdditionalHomeScreens: (paramList) => {
```

Comment:
I was planning to test this in particular more to be sure because I've only gone as far as making it pass tests. I also have some doubts that you can simply add screens from a different stack.

Reply:
I tested this out and everything works as expected:
```
export const getAdditionalHomeScreens: ExpandedDeveloperConfig['getAdditionalHomeScreens'] =
  (paramList) => {
    const HomeStack = createNativeStackNavigator<typeof paramList>();
    return [
      <HomeStack.Screen
        key="SettingsStack"
        name="SettingsStack"
        component={SettingsStack}
        options={{ headerShown: false }}
      />,
      <HomeStack.Screen
        key="NotificationsStack"
        name="NotificationsStack"
        component={NotificationsStack}
        options={{ headerShown: false }}
      />,
    ];
  };
  ```